### ISSUE TITLE
val-acc always at~10% while the training loss can be reduced and the training accuracy can be increased

### ISSUE THREAD

**Issue by Kyrie10favor** [megvii-model/AngleNAS#13]
-   

Hello, I used my own dataset, which has 10 categories.at first ,I finished the **Shrinking** in SPOS folder ,and then I entered ProxyLess folder in order to Search over the Shrunk Search Space .However I found the problem:**`val-Top1acc always about~10%,val-Top5acc always about~50%  while the training loss can be reduced and the training accuracy can be increased`** ，So It seems to be the result of a random prediction on the validation set。
 

-  How can I solve this problem?thank u.
